docs/admin/executors/deploy_executors_kubernetes.mdx

```diff
@@ -1,6 +1,6 @@
 # Deploying Sourcegraph executors natively on Kubernetes
 
-<Callout type="note">This feature is in beta and is available in Sourcegraph 5.2.3 and later.</Callout>
+<Callout type="note">This feature is in beta and is available in Sourcegraph 5.1.0 and later.</Callout>
```

Comment:
I'm not sure why the automation updates this version number... Reverted this back to v5.1.0, which was the version where support for k8s executors was introduced

Reply:
Hmm yea likely a misnomer